cvat/apps/engine/static/engine/js/player.js

```diff
@@ -123,6 +123,7 @@ class FrameProvider extends Listener {
 
 const MAX_PLAYER_SCALE = 10;
 const MIN_PLAYER_SCALE = 0.1;
+const PLAYER_FRAME_OFFSET = 1000;
```

Comment:
PLAYER_FRAME_OFFSET doesn't look as a good solution. It is a workaround for mention problems but may be it is a whole in GUI design. Let's find a solution without PLAYER_FRAME_OFFSET.

Reply:
One more possible solution is to emphasize the drawing area visually (white paper).

Reply:
Done